sonar-helm-for-iac/org.sonarsource.iac.helm/io-handling.go

```diff
@@ -0,0 +1,74 @@
+package org_sonarsource_iac_helm
```

Comment:
IMO We should use this package only for generated files and the rest should be in sonar-helm-for-iac or in other package

Reply:
I don't really like the mix of pom.xml, scripts, Go files etc. in the root package; I would prefer to split the Go code more into subpackages. I'll try to rename it then